src/kudu/consensus/raft_consensus.h

```diff
@@ -119,6 +119,7 @@ class RaftConsensus : public std::enable_shared_from_this<RaftConsensus>,
                       public PeerMessageQueueObserver {
  public:
   typedef std::function<void(const ElectionResult&)> ElectionDecisionCallback;
+  typedef std::function<void(void)> TermAdvancementCallback;
```

Comment:
Wondering if it makes sense to pass the new term as an argument to the callback?